flexget/plugins/api_trakt.py

```diff
@@ -147,136 +475,212 @@ class TraktSearchResult(Base):
+            return
+        except plugin.PluginError as e:
+            console('Authorization failed: %s' % e)
+    elif options.action == 'show':
+        if not options.account:
+            console('Please specify an account')
+            return
+        with Session() as session:
+            acc = session.query(TraktUserAuth).filter(TraktUserAuth.account == options.account).first()
+            if acc:
+                console('Authorization expires on %s' % acc.expires)
+            else:
+                console('Flexget has not been authorized to access your account.')
+    elif options.action == 'refresh':
+        if not options.account:
+            console('Please specify an account')
+            return
+        try:
+            get_access_token(options.account, refresh=True)
+            console('Successfully refreshed your access token.')
+            return
+        except plugin.PluginError as e:
+            console('Authorization failed: %s' % e)
 
-            log.debug('Episode %s not found in cache, looking up from trakt.' % ep_description)
-            try:
-                response = requests.get(url)
-            except requests.RequestException:
-                raise LookupError('Error Retrieving Trakt url: %s' % url)
-            try:
-                data = response.json()
-            except ValueError:
-                raise LookupError('Error parsing Trakt episode json for %s' % title)
-            if not data:
-                raise LookupError('No data in response from trakt %s' % url)
-            if 'error' in data:
-                raise LookupError('Error looking up %s: %s' % (ep_description, data['error']))
-            ep_data = data['episode']
-            if not ep_data:
-                raise LookupError('No episode data %s' % url)
-            episode = session.query(TraktEpisode).filter(TraktEpisode.tvdb_id == ep_data['tvdb_id']).first()
-            if not episode:
-                # Update result into format TraktEpisode expects (?)
-                ep_data['episode_name'] = ep_data.pop('title')
-                ep_data.update(ep_data['images'])
-                del ep_data['images']
-
-                episode = TraktEpisode(ep_data)
-                series.episodes.append(episode)
-                session.merge(episode)
 
-        return episode
+@event('options.register')
+def register_parser_arguments():
+    # Register subcommand
+    parser = options.register_command('trakt', do_cli, help='view and manage trakt authentication.'
+                                                            'Please visit %s to retrieve your pin code.' % PIN_URL)
+    # Set up our subparsers
+    subparsers = parser.add_subparsers(title='actions', metavar='<action>', dest='action')
+    auth_parser = subparsers.add_parser('auth', help='authorize Flexget to access your Trakt.tv account')
+
+    auth_parser.add_argument('--account', metavar='<account>', help='local identifier')
```

Comment:
Maybe add something like 'this is how you will refer to these credentials from your flexget config'